tracer/src/Datadog.Trace/Telemetry/TelemetryController.cs

```diff
@@ -138,6 +142,52 @@ public void SetFlushInterval(TimeSpan flushInterval)
         _queue.Enqueue(new WorkItem(WorkItem.ItemType.SetFlushInterval, flushInterval));
     }
 
+    public async Task DumpTelemetry(string filePath)
+    {
+        if (!_isStarted)
+        {
+            // we haven't performed all startup initialization yet, so we can't dump anything
+            Log.Information("Unable to dump telemetry as controller not yet initialized");
+            return;
+        }
+
+        try
+        {
+            var application = _application.GetApplicationData();
+            var host = _application.GetHostData();
+            if (application is null || host is null)
+            {
+                Log.Debug("Telemetry not initialized, skipping");
```

Comment:
Should be info IMO

Reply:
It'll get written either way, because we enable debug just before, but sure

Reply:
Oops, I merged before I fixed this 😬 I'll include it as a tweak in a separate PR to avoid a whole new CI run 😅